Makefile

```diff
@@ -1,18 +1,18 @@
 # Current Operator version
-VERSION ?= 2.0.0-snapshot
+VERSION ?= 7.11.0
 # Default bundle image tag
-BUNDLE_IMG ?= quay.io/kiegroup/kogito-cloud-operator-bundle:$(VERSION)
+BUNDLE_IMG ?= quay.io/kiegroup/rhpam-kogito-operator-bundle:$(VERSION)
 # Default catalog image tag
-CATALOG_IMG ?= quay.io/kiegroup/kogito-cloud-operator-catalog:$(VERSION)
+CATALOG_IMG ?= quay.io/kiegroup/rhpam-kogito-operator-catalog:$(VERSION)
 # Options for 'bundle-build'
-CHANNELS=alpha,1.x
+CHANNELS=7.x
```

Comment:
I think we will continue to use 1.x;  let's see.

Reply:
same here.
https://github.com/kiegroup/rhpam-kogito-operator/pull/2#discussion_r595336240